module/config.mjs

```diff
@@ -1805,9 +1850,15 @@ DND5E.encumbrance = {
       m: 1.5
     }
   },
-  vehicleWeightMultiplier: {
-    imperial: 2000, // 2000 lbs in an imperial ton
-    metric: 1000 // 1000 kg in a metric ton
+  baseUnits: {
+    default: {
+      imperial: "lb",
+      metric: "kg"
+    },
+    vehicle: {
+      imperial: "tn",
+      metric: "mg"
```

Comment:
Using "mg" here so presumably just a mistake up above.

Reply:
What mistake are you referring to?

Reply:
Using "t" as the abbreviation for Megagrams in the localization.

Reply:
This is the static `key`, not the localization value, so it references the key in `CONFIG.weightUnits`.